composer.json

```diff
@@ -31,5 +31,10 @@
         "psr-4": {
             "MaxMind\\Db\\": "src/MaxMind/Db"
         }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "MaxMind\\Db\\Test\\Reader\\": "tests/MaxMind/Db/Test/Reader"
```

Comment:
If we do this, I think we can get rid of `./tests/bootstrap.php` and `phpunit.xml.dist` to use `./vendor/autoload.php` instead.

Reply:
Yes. You're right. The request change is accepted :).